tests/test_fl.py

```diff
@@ -513,10 +519,293 @@ def test_chunk_name_limit(tmp_path, open_mode):
                      application='test_chunk_name_limit',
                      schema='none',
                      schema_version=[1, 2]) as f:
-        for i in range(128):
+        for i in range(65535):
             f.write_chunk(name=str(i), data=numpy.array([i], dtype=numpy.int32))
 
-        # A bug in GSD limits files to 128 chunk names:
-        # see https://github.com/glotzerlab/gsd/issues/43
+        # The GSD specification limits to 65535 names:
         with pytest.raises(Exception):
```

Comment:
To be specific, this raises a RuntimeError, right?

Reply:
Yes, fixed.